developers_chamber/ecs_utils.py

```diff
@@ -193,25 +193,64 @@ def start_service(cluster, service, count, region):
     except ecs_client.exceptions.ClusterNotFoundException:
         raise ClickException("Cluster not found: '{}'".format(cluster))
     except ecs_client.exceptions.ServiceNotFoundException:
-        raise ClickException("Service not found: '{}'".format(cluster))
+        raise ClickException("Service not found: '{}'".format(service))
     except ClientError as ex:
         raise ClickException(ex)
 
 
-def start_services(cluster, count, region):
+def start_services(cluster, services, count, region):
```

Comment:
Mám dojem, že podstatné bloky fcí `start_services` a `start_cluster_services` jsou v podstatě identické - nastálo by za to je vytáhnout někam ven do společné funkce?

Reply:
A blok s Loggerem + try/except je víceméně identický i pro fci `stop_services` - liší se jenom logovaným textem, ale ten by šel do společné fce předat zvenku jako parametr.

Reply:
> Mám dojem, že podstatné bloky fcí `start_services` a `start_cluster_services` jsou v podstatě identické - nastálo by za to je vytáhnout někam ven do společné funkce?

No vidíš to mi v tu chvíli nedošlo 👍

Reply:
> A blok s Loggerem + try/except je víceméně identický i pro fci `stop_services` - liší se jenom logovaným textem, ale ten by šel do společné fce předat zvenku jako parametr.

Díky popřemýšlím jak to vylepšit aby se to tolik neopakovalo 👍

Reply:
vyřešeno
`start_cluster_services` volá `start_services`